test/models/order_test.rb

```diff
@@ -32,6 +32,7 @@ def setup
   end
 
   test "order has items associated with it" do
-    skip
+    item = order.items.create(title: 'coffee')
+    assert_equal 'coffee', order.items.first.title
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.